google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2922,6 +2962,40 @@ boolean isValid() {
     }
   }
 
+  /**
+   * Returns a multiplexed session. We would always return the session which is at the front of the
+   * queue since this will be the most recently created session.
```

Comment:
nit: this comment is a bit out of date, as this implementation no longer uses a queue

Reply:
Reworded this docs.